### DISCUSSION TITLE
While accessing Grammer getting module erro

### DISCUSSION THREAD

**Discussion by krishnaisdinesh** [explosion/spaCy#7787] [Help: Installation] [answered]
Code : 
import spacy
from spacy_grammar.grammar import Grammar

nlp = spacy.load('en')
grammar = Grammar(nlp)
nlp.add_pipe(grammar)
doc = nlp('I can haz cheeseburger.')
doc._.has_grammar_error  # True

//////////////////////////
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-8-076ff3ef6037> in <module>
      1 import spacy
----> 2 from spacy_grammar.grammar import Grammar
      3 
      4 nlp = spacy.load('en')
      5 grammar = Grammar(nlp)

ModuleNotFoundError: No module named 'spacy_grammar'

## Your Environment

- Operating System: Ubuntu 
- Python Version Used: 3
- spaCy Version Used: 
- Environment Information:

**Comment by polm** [comment, marked as answer]
It looks like [spacy_grammar](https://github.com/tokestermw/spacy_grammar) is an unofficial third-party module for spaCy 2 that hasn't been updated in four years. 

You would get that error if you haven't installed the package. You need to follow the instructions in the package README. It's not on pip so you have to check out the git repo first.

**Reply by krishnaisdinesh** [reply]
Following this updated one and didn't fount anything to install seperately.

https://spacy.io/universe/project/spacy_grammar

**Reply by polm** [reply]
That's the same project. spaCy Universe is a directory of community projects, not features included in the base spaCy package.

**Reply by krishnaisdinesh** [reply]
spaCy version    3.0.5                         
Python version   3.8.5                         
Pipelines        en_core_web_sm (3.0.0)

**Reply by svlandeg** [reply]
Unfortunately, as Paul pointed out, it looks like `spacy_grammar` hasn't been recently maintained, and isn't supporting spaCy v3.
